### PR TITLE
Install theme scss

### PR DESCRIPTION
**Pull request by EmmanuelDrouin**
- Install theme 2.0
- Import theme into 'structure', 'variables' and 'style'. Remove useless elements.
- Update all variables labels

### CODE REVIEW

**Comment by EmmanuelDrouin** [comment]
Did a mistake, PR linked to the #209

File: components/List.vue [hunk 1]
```diff
@@ -19,6 +19,7 @@ export default {
 </script>
 
 <style lang="scss" scoped>
+@import "~/assets/_variables";
```
Comment (antho1404) [inline components/List.vue]: can be removed, it doesn't use any variables

File: pages/faq.vue [hunk 2]
```diff
@@ -102,6 +102,7 @@ export default {
 </script>
 
 <style lang="scss" scoped>
+@import "~/assets/_variables";
```
Comment (antho1404) [inline pages/faq.vue]: can be removed, it doesn't use any variables

File: components/List.vue (hunk 1 above)
Comment (antho1404) [inline components/List.vue]:
```suggestion
```

File: pages/faq.vue (hunk 2 above)
Comment (antho1404) [inline pages/faq.vue]:
```suggestion
```

**Pull request merged**